### PR TITLE
CLOUDP-219313: Document test kube version

### PR DESCRIPTION
**Pull request by josvazg**
### All Submissions:

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: docs/dev/ci.md [hunk 1]
```diff
@@ -40,3 +40,20 @@ The `test.yml` workflow calls the local tests workflows directly, and then also
 That workflow also shows relevant CI context values that allow us to debug why the CI took one decision or another, depending of whether the PR is a **draft**, it is **forked or not**, etc.
 
 The `test.yml` takes the output from `cloud-tests-filter.yml` and will ONLY invoke the `cloud-tests.yml` workflow IF `cloud-tests-filter.yml` had decided **Cloud Tests* should run.
+
+## Kubernetes version in CI tests
+
+The kubernetes version in CI tests is set purposefully to the oldest kubernetes version we need to support.
+
+Such version is set by parameterising the kind image tag within the **strategy** **matrix** at the `test-e2e.yml` workflow. Eg:
```
Comment (s-urbaniak) [inline docs/dev/ci.md:48]:
```suggestion
Such version is set by parametrising the kind image tag within the **strategy** **matrix** at the `test-e2e.yml` workflow. Eg:
```

**Review by s-urbaniak — COMMENTED** [review]
one nit, else LGTM 👍

File: docs/dev/ci.md (hunk 1 above)
Comment (josvazg) [reply docs/dev/ci.md:48]: for "parametrising" Google tells me "Did you mean: [parameterising](https://www.google.com/search?sca_esv=595327113&rlz=1C5GCEM_enES1061ES1061&q=parameterising&spell=1&sa=X&sqi=2&ved=2ahUKEwiCp6PP_8CDAxVIXvEDHSgWDtAQBSgAegQIBhAC&biw=2472&bih=1294&dpr=1)"

Comment (s-urbaniak) [reply docs/dev/ci.md:48]: oh, TIL ! 👍

**Review by s-urbaniak — APPROVED** [review]
LGTM

**Pull request merged**